definitions/divorce/json/AuthorisationCaseEvent.json

```diff
@@ -4527,5 +4534,208 @@
     "CaseEventID": "cleanCaseState",
     "UserRole": "caseworker-divorce-courtadmin-la",
     "CRUD": "CU"
+  },
+  {
+    "LiveFrom": "25/06/2019",
```

Comment:
I think these shouldn't go to prod, should they?

Reply:
I mean, the solicitor journey ones...

Reply:
The Read permissions on events are fine to go to prod.